lighthouse-core/config/default.js

```diff
@@ -5,10 +5,14 @@
  */
 'use strict';
 
+const Driver = require('../gather/driver');
```

Comment:
I'm not a fan of requiring core files in default config, but we'll need a strategy for this when it comes to throttling soon. split constants out to a separate file? have them canonically live in config?

Reply:
if those settings like max_wait_for_fully_loaded will live in settings they can probably be removed from the original file so why not put them in here to start with?

Reply:
well that's the question up for discussion :)

it makes some sense to continue having them colocated with the logic they affect as well as to simplify unit testing where a full default config is not given (think the lantern simulation settings for example)

Reply:
+1 for putting these constants in the config. Requiring the driver from cli-flags was the absolute worst. Basing them in config seems most natural.

Reply:
@patrickhulce yeah +1 for config